components/brave_rewards/browser/test/common/rewards_browsertest_contribution.h

```diff
@@ -153,8 +154,9 @@ class RewardsBrowserTestContribution
   ledger::type::Result ac_reconcile_status_ =
       ledger::type::Result::LEDGER_ERROR;
 
-  Browser* browser_;  // NOT OWNED
-  brave_rewards::RewardsServiceImpl* rewards_service_;  // NOT OWNED
+  raw_ptr<Browser> browser_ = nullptr;  // NOT OWNED
+  raw_ptr<brave_rewards::RewardsServiceImpl> rewards_service_{
```

Comment:
Nit: `= nullptr` vs. `{nullptr}` - any reason for both?

Reply:
cc: @mariospr

Reply:
Oops! No good reason, it seems to me that I used the wrong replacement in the regexp-based search&replace process for some cases, and those made it to the final commit when squashing my local commits into the PR.

Fixed in https://github.com/brave/brave-core/commit/c5e9e215f54b8805fc89f7117472f87d6ca15c7d